cmd/reconciler/main.go

```diff
@@ -95,25 +96,30 @@ var (
 		"Enable debug mode, panicking in many scenarios where normally an InternalError would be logged. "+
 			"Do not use in production.")
 
-	renderingEnabled = flag.Bool("rendering-enabled", util.EnvBool(reconcilermanager.RenderingEnabled, false), "")
+	renderingEnabled  = flag.Bool("rendering-enabled", util.EnvBool(reconcilermanager.RenderingEnabled, false), "")
+	namespaceStrategy = flag.String(flags.namespaceStrategy, util.EnvString(reconcilermanager.NamespaceStrategy, ""),
```

Comment:
Why isn't the default set here?

Reply:
I opted to follow the existing precedent in this file and set the default in the main function below. Would you prefer I refactor this to set the default value up here?

Reply:
Yes, it's weird not to set the default directly where the flag is defined but set it separately. Setting default is already supported in the `flag.*` functions.